<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalizedStringEditorModal</class>
 <widget class="QDialog" name="LocalizedStringEditorModal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="1">
    <widget class="QLineEdit" name="fieldKorean"/>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="fieldChineseTrad"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="fieldSpanish"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Mexican</string>
     </property>
     <property name="buddy">
      <cstring>fieldMexican</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Japanese</string>
     </property>
     <property name="buddy">
      <cstring>fieldJapanese</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="fieldFrench"/>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Polish</string>
     </property>
     <property name="buddy">
      <cstring>fieldPolish</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>French</string>
     </property>
     <property name="buddy">
      <cstring>fieldFrench</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="fieldEnglish"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Chinese (Traditional)</string>
     </property>
     <property name="buddy">
      <cstring>fieldChineseTrad</cstring>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLineEdit" name="fieldPolish"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="fieldJapanese"/>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="fieldChineseSimp"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Chinese (Simplified)</string>
     </property>
     <property name="buddy">
      <cstring>fieldChineseSimp</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Italian</string>
     </property>
     <property name="buddy">
      <cstring>fieldItalian</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Portugese</string>
     </property>
     <property name="buddy">
      <cstring>fieldPortugese</cstring>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="fieldMexican"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>German</string>
     </property>
     <property name="buddy">
      <cstring>fieldGerman</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLineEdit" name="fieldPortugese"/>
   </item>
   <item row="14" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSaveAsNew">
       <property name="text">
        <string>Save as New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Korean</string>
     </property>
     <property name="buddy">
      <cstring>fieldKorean</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="fieldGerman"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>English</string>
     </property>
     <property name="buddy">
      <cstring>fieldEnglish</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Spanish</string>
     </property>
     <property name="buddy">
      <cstring>fieldSpanish</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="fieldItalian"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="labelStringIndex">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>String table entry #%1</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fieldEnglish</tabstop>
  <tabstop>fieldJapanese</tabstop>
  <tabstop>fieldGerman</tabstop>
  <tabstop>fieldFrench</tabstop>
  <tabstop>fieldSpanish</tabstop>
  <tabstop>fieldMexican</tabstop>
  <tabstop>fieldItalian</tabstop>
  <tabstop>fieldKorean</tabstop>
  <tabstop>fieldChineseTrad</tabstop>
  <tabstop>fieldChineseSimp</tabstop>
  <tabstop>fieldPortugese</tabstop>
  <tabstop>fieldPolish</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
